sdk/keyvault/keyvault-secrets/src/lro/recover/operation.ts

```diff
@@ -50,7 +44,7 @@ export class RecoverDeletedSecretPollOperation extends KeyVaultSecretPollOperati
    * This operation requires the secrets/get permission.
    */
   private getSecret(name: string, options: GetSecretOptions = {}): Promise<KeyVaultSecret> {
-    return withTrace("getSecret", options, async (updatedOptions) => {
+    return tracingClient.withSpan("SecretClient.getSecret", options, async (updatedOptions) => {
```

Comment:
Can we name this `SecretPoller` similar to how you are naming other spans in pollers above